tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -125,6 +125,7 @@ protected:
    struct RSharedPtrDeleter {
       std::unique_ptr<RFieldBase::RDeleter> fDeleter;
       void operator()(void *objPtr) { fDeleter->operator()(objPtr, false /* dtorOnly*/); }
+      RSharedPtrDeleter(const RSharedPtrDeleter& deleter) : fDeleter( new RFieldBase::RDeleter(*deleter.fDeleter)) {}
```

Comment:
@vepadulano - this change can be ignored. I need it to build root on my MacOS 11.6.

Reply:
removed